arangod/IResearch/IResearchLink.cpp

```diff
@@ -2379,6 +2353,35 @@ void IResearchLink::LinkStats::toPrometheus(std::string& result,       //
   _needName = false;
 }
 
+void IResearchLink::LinkStats::toVPack(
+    application_features::ApplicationServer& server,
+    velocypack::Builder& builder, std::string_view labels) const {
+  auto const start = labels.find(",shard=\"");
```

Comment:
If for whatever reason the `find(",shard=\"");` returns `std::string_view::npos`, then the following code can fail.
We should add a check here that `start != std::string_view::npos`!

Reply:
I can add tri_assert